app/javascript/readingList/readingList.jsx

```diff
@@ -73,11 +91,11 @@ export class ReadingList extends Component {
       : READING_LIST_PATH;
 
     // empty items so that changing the view will start from scratch
-    this.setState({ statusView: newStatusView, items: [] });
+    this.setState({ statusView: newStatusView, items: [], selectedTag });
 
     this.search(query, {
       page: 0,
-      tags: selectedTags,
+      tags: [],
```

Comment:
Although we're filtering by one tag now, the search query still requires an array which is why this gets reset to an empty array.